Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -567,6 +564,30 @@ static void logProcessPoolState(const WebProcessPool& pool)
     unregisterNotificationObservers();
 }
 
+String WebProcessPool::presentingBundleIdentifierFromAuditToken(audit_token_t audit_token)
+{
+#if PLATFORM(MAC)
+    LSSessionID sessionID = (LSSessionID)audit_token_to_asid(audit_token);
+    auto auditTokenDataRef = adoptCF(CFDataCreate(kCFAllocatorDefault, (const UInt8 *)(&audit_token), sizeof(audit_token)));
```

Comment:
Nit - would it be possible to share some of this code with `ClientConnection::bundleIdentifierFromAuditToken`? (Perhaps, as a separate helper function in `WebKit/Shared`)

Reply:
That's a good idea, let me extract this logic into a shared helper function.

Reply:
It turns out I was overcomplicating this. Since the code here runs inside the UI process, I've updated this PR to use NSRunningApplication in order to get the expected bundle identifier.

The code in webpushd still need to use the more complicated lookup since it runs out of process, so I'll leave that unchanged.